### PR TITLE
feat(edgeless): support DnD images from outside into editor

### LINKED ISSUE #1447: Support dragging and dropping single or multiple images from outside the browser into the editor for adding
This is a very natural way to add local images to the AFFiNE editor. We hope it can be supported based on feedback from the community.

### PR DESCRIPTION
**Pull request by fundon**
Based on #1487 
Closes #1447

### CODE REVIEW

**Review by doodlewind — COMMENTED** [review]
This generally looks good, I'd just add some minor style updates to get it merges. @fundon could you help getting the conflicts resolved? Thanks!

**Comment by fundon** [comment]
> @fundon could you help getting the conflicts resolved?

Resolved.

File: packages/editor/src/components/editor-container.ts [hunk 1]
```diff
@@ -69,6 +90,79 @@ export class EditorContainer
     pageModeSwitched: new Slot(),
   };
 
+  // add files from outside
+  private async _onDropEnd(
```
Comment (doodlewind) [inline packages/editor/src/components/editor-container.ts:94]: This should be better contained in the `FileDropManager`, the `EditorContainer` should be kept as a thin wrapper.

File: packages/editor/src/components/editor-container.ts [hunk 2]
```diff
@@ -69,6 +90,79 @@ export class EditorContainer
...
+        assertExists(note);
+        model = note.model.lastItem();
+      }
+    }
+
+    if (type === 'database') {
+      type = 'after';
+    }
+
+    let noteId;
```
Comment (doodlewind) [inline packages/editor/src/components/editor-container.ts:123]: It's preferred to use `const` over `let` and these variables also lacks typing.

**Review by doodlewind — APPROVED** [review]
Thanks for the feat! Let's land this.

**Pull request merged**